packages/query/src/query-engine/index.ts

```diff
@@ -124,40 +58,17 @@ export class QueryEngine {
 		  }
 		| undefined
 	> => {
-		if ( ! QueryEngine.isClientInitialized ) {
-			QueryEngine.initialize();
-		}
-
-		try {
-			const data = await QueryEngine.apolloClient.query( {
-				query: GetGeneralSettingsDocument,
-				fetchPolicy: 'network-only', // @todo figure out a caching strategy, instead of always fetching from network
-				errorPolicy: 'all',
-			} );
-
-			return parseGeneralSettings( data );
-		} catch ( error ) {
-			if ( error instanceof ApolloError ) {
-				logApolloErrors( error );
-
-				// If there are networkError throw the error with proper message.
-				if ( error.networkError ) {
-					// Throw the error with proper message.
-					throw new Error(
-						getNetworkErrorMessage( error.networkError )
-					);
-				}
-			}
+		const data = await fetchQuery( {
+			name: [ 'generalSettings' ],
```

Comment:
Names should be the GraphQL name
```suggestion
			name: 'GetGeneralSettings',
```